lib/pages/select_language.dart

```diff
@@ -0,0 +1,143 @@
+        child: SafeArea(
+          bottom: false,
+          child: Padding(
+            padding: const EdgeInsets.only(left: 14, right: 14.0, top: 25),
+            child: ListView.separated(
+              itemCount: languageCode.length,
+              itemBuilder: (context, index) {
+                return languageTile(
+                  languageCode: languageCode[index],
+                  index: index,
+                  length: languageCode.length,
+                  context: context,
+                );
+              },
+              separatorBuilder: (context, index) => Divider(
+                color: Color(0xffefeef3),
+                thickness: 0.3,
+                height: 2,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget languageTile({
+    required String languageCode,
+    required int index,
+    required int length,
+    required BuildContext context,
+  }) {
+    return GestureDetector(
+        child: Container(
+          height: 48,
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.only(
+              topLeft: index == 0 ? Radius.circular(12.0) : Radius.circular(0),
+              topRight: index == 0 ? Radius.circular(12.0) : Radius.circular(0),
+              bottomLeft: index == length - 1
+                  ? Radius.circular(12.0)
+                  : Radius.circular(0),
+              bottomRight: index == length - 1
+                  ? Radius.circular(12.0)
+                  : Radius.circular(0),
+            ),
+            color: Colors.white,
+          ),
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 14.0),
+            child: Row(
+              children: [
+                Text(
+                  getLenguageString(
+                      languageCode: languageCode, context: context),
+                  style: TextStyle(fontSize: 20),
+                ),
+                Spacer(),
+                BlocBuilder<LanguageCubit, LanguageState>(
+                  bloc: Get.find<LanguageCubit>(),
+                  builder: (context, state) {
+                    if (state is NewLanguage) {
+                      if (state.language == languageCode) {
+                        return Icon(
+                          Icons.check_circle_rounded,
+                          size: 26,
+                          color: Color(0xFF004DFF),
+                        );
+                      } else {
+                        return SizedBox.shrink();
+                      }
+                    } else {
+                      return SizedBox.shrink();
+                    }
+                  },
+                )
+              ],
+            ),
+          ),
+        ),
+        onTap: () async {
+          Get.find<LanguageCubit>().changeLanguage(language: "$languageCode");
+          Get.updateLocale(Locale("$languageCode"));
+        });
+  }
+
+  String getLenguageString(
```

Comment:
@huynguyennovem I thought about it, but it will be a little more difficult in terms of language code translations, I will probably add this a little bit later with the design update with Dmitry